tests/test_attachments.py

```diff
@@ -270,3 +270,48 @@ async def default_handler(message: IncomingMessage, bot: Bot) -> None:
     # - Assert -
     assert incoming_message
     assert incoming_message.file == domain_attachment
+
+
+API_UNSUPPORTED_FILE_ATTACHMENT_TYPE = (
+    {
+        "data": {
+            "id": "9657694f-166e-575a-b6ea-6575521d4f57",
+            "pack": "2c00198f-6efb-5b36-bed1-1a380bc85710",
+            "version": 1637674405,
+        },
+        "type": "sticker",
+    },
+)
+
+
+@pytest.mark.parametrize(
+    "api_attachment",
+    API_UNSUPPORTED_FILE_ATTACHMENT_TYPE,
+)
+async def test__async_execute_raw_bot_command__unsupported_attachment_types(
+    api_attachment: Dict[str, Any],
+    api_incoming_message_factory: Callable[..., Dict[str, Any]],
+    bot_account: BotAccountWithSecret,
+    loguru_caplog: pytest.LogCaptureFixture,
+) -> None:
+    # - Arrange -
+    payload = api_incoming_message_factory(attachment=api_attachment)
+
+    collector = HandlerCollector()
+    incoming_message: Optional[IncomingMessage] = None
+
+    @collector.default_message_handler
+    async def default_handler(message: IncomingMessage, bot: Bot) -> None:
+        nonlocal incoming_message
+        incoming_message = message
+        # Drop `raw_command` from asserting
+        incoming_message.raw_command = None
```

Comment:
You can drop this lines, 'cause you don't asserting `incoming_message`.

Reply:
Fix.